### ISSUE TITLE
Remove old manifests

### ISSUE THREAD

**Issue by enclave-marc-barry** [enclave-networks/install.enclave.io#6]
Our manifest file format has evolved and led to old files and schemas in different places and repos. This issue exists to track clean-up of the old files.

##  Remove `/latest` directory from enclave.io website

This was the original location of manifest files. The only system dependant on this location is the Enclave Synology NAS drive in the Newport office. Its running an old (2019) version of Enclave using Mono and cannot be upgraded to the more recent versions of Enclave which target native complication and do not depend on mono. (see [this issue](https://github.com/enclave-networks/product-management/issues/32))

I've removed all old install scripts and all manifests except for the Linux manifest (GUID `enclave-networks.github.io\latest\09fdd2f5-dcbf-427f-8195-a1b5004c9ab9.html`).

If this file is removed, the version of Enclave running on the NAS will refuse to start. One possibility is to run Enclave on the NAS using Docker.

## Remove `install.enclave.io\latest\` directory and contained all files

When moving to the `install.enclave.io` repo, we originally preserved the concept of `/latest` along with manifests named with guids. The current GA release (`2020.10.27.0`) depends on this location so we cannot remove it until agents are updated to any release of enclave dated later than 20th November 2020.

Also in this directory are latest files, `version` and `preprod-version`. There is another issue raised to remove these files and operate a single source of truth for the latest product version (i.e. the manifest files only).

Once both dependencies are resolved this directory can be removed.

**Comment by enclave-alistair** [comment]
Is the Synology NAS still a problem @enclave-marc-barry ?

**Comment by enclave-marc-barry** [comment]
Nope, I believe <redacted> may also be on older versions that prevent the removal of this file.
